xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-filters/xwiki-platform-notifications-filters-api/src/main/java/org/xwiki/notifications/filters/internal/scope/ScopeNotificationFilterPreference.java

```diff
@@ -161,7 +161,7 @@ public boolean isEnabled()
     @Override
     public boolean isActive()
     {
-        return filterPreference.isActive();
+        return true;
```

Comment:
Maybe this should be the default implementation right in the interface, so that no class implementing NotificationFilterPreference ever have to take care of it.